spec/support/helpers/auth_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module AuthHelper
+  ACCESS_TOKEN = Struct.new(:access_token).new('an-access-token')
+
+  def stub_auth_token
+    allow(Nomis::Oauth::TokenService).to receive(:valid_token).and_return(ACCESS_TOKEN)
+
+    stub_request(:post, "https://gateway.t3.nomis-api.hmpps.dsd.io/auth/oauth/token?grant_type=client_credentials").
```

Comment:
Is this URL correct? Should it be stubbing the new URL, rather than the old T3 URL?

```suggestion
    stub_request(:post, "https://sign-in-dev.hmpps.service.justice.gov.uk/auth/oauth/token?grant_type=client_credentials").
```